man/demo_loanbook.Rd

```diff
@@ -19,7 +19,7 @@ demo_loanbook
 }
 \description{
 A simplified demo tilt company data set to illustrate and test matching with
-loanbook. For details on included cases please refer to data generation
+loanbook. For details on included cases please refer to the data generation
```

Comment:
If you want to change this please change in the source (R/data_demo_loanbook.R) and rerender the documentation mit devtools::document() :).
Since this is only a test PR please feel free to close it instead